depends/packages/packages.mk

```diff
@@ -1,6 +1,6 @@
 rust_packages := rust librustzcash
 zcash_packages := libsnark libgmp libsodium
-packages := boost openssl zeromq $(zcash_packages) googletest googlemock
+packages := boost openssl zeromq $(zcash_packages) googletest googlemock proton
```

Comment:
This is going to conflict with #2176, but it also makes sense for the dependency to be here, so whichever one goes second will have to rebase :)